master/usage/policy/secure-metrics.md

```diff
@@ -0,0 +1,415 @@
+---
+title: Securing access to Calico's metrics endpoints
+---
+
+## About securing access to {{site.prodname}}'s metrics endpoints
+
+When using {{ site.prodname }} with prometheus metrics enabled, it may be desirable to use network policy
+to limit access to {{ site.prodname }}'s metrics endpoints.
+
+### Purpose of this page
```

Comment:
I think the Purpose sections are a little overly formal...you could just make this one sentence and put it into the previous About section